internal/oci/container.go

```diff
@@ -25,8 +26,9 @@ import (
 const defaultStopSignalInt = 15
 
 var (
-	defaultStopSignal   = strconv.Itoa(defaultStopSignalInt)
-	ErrContainerStopped = errors.New("container is already stopped")
+	defaultStopSignal      = strconv.Itoa(defaultStopSignalInt)
+	ErrContainerStopped    = errors.New("container is already stopped")
+	errContainerPidStopped = errors.Errorf("insufficient ps information; pid likely stopped")
```

Comment:
1. `s/errors.Errorf/errors.New/`
2. "pid is stopped" seems confusing to me (I think of it as if the process was stopped by SIGSTOP). Something like "no such pid" or "process unexpectedly gone" would be better.

Reply:
and the error name itself should be `errContainerInitGone` or something similar (`errContainerInitPidNotFound`?)